src/docs/references/transaction-process-actions/index.md

```diff
@@ -799,6 +799,120 @@ Decline a pending or proposed booking.
 
 **Configuration options**: -
 
+### Stock reservations
+
+#### :action/create-pending-stock-reservation
+
+Creates a new stock reservation in `pending` state with given stock, as
+long as the listing's stock permits the reservation to be created.
+
+A stock reservation in `pending` state decreases the listing's stock - a
+stock adjustment is created having the given `stockReservationQuantity`
+(with a negative sign).
+
+There can be at most one stock reservation per transaction.
+
+**Preconditions**:
+
+- The transaction must not already have a stock reservation.
+- The listing must have sufficient stock available.
+
+**Parameters**:
+
+- `stockReservationQuantity`: positive integer, mandatory.
+
+  The quantity of stock that is reserved from a listing's stock.
+
+**Configuration options**: -
+
+#### :action/create-proposed-stock-reservation
+
+Creates a new stock reservation in `proposed` state with given stock, as
+long as the listing's stock permits the reservation to be created.
+
+A stock reservation in `proposed` state does not affect the listing's
+available stock, i.e. the listing's stock is not decreased by the
+proposed reservation quantity until the reservation is accepted using
+the `:action/accept-stock-reservation` action.
+
+There can be at most one stock reservation per transaction.
+
+**Preconditions**:
+
+- The transaction must not already have a stock reservation.
+- The listing must have sufficient stock available.
+
+**Parameters**:
+
+- `stockReservationQuantity`: positive integer, mandatory.
+
+  The quantity of stock for the reservation.
+
+**Configuration options**: -
+
+#### :action/accept-stock-reservation
+
+Mark stock reservation as accepted.
+
+A stock reservation in `accepted` state decreases the listing's stock.
+That means that if the stock reservation was previously in a `proposed`
+state, a new stock adjustment is created with the quantity of the stock
+reservation (with a negative sign).
+
+**Preconditions**:
+
+- The transaction must have a stock reservation
+- The stock reservation must be in either `proposed` or `pending` state
+- If the stock reservation is in `proposed` state, the listing must have
+  sufficient stock so that the stock reservation can be accepted.
+
+**Parameters**:
+
+**Configuration options**: -
+
+#### :action/decline-stock-reservation
+
+Decline a pending or proposed stock reservation.
+
+If the stock reservation was previously in a `pending` state, a new
+stock adjustment is created for the listing, reversing the adjustmet
+that resulted from the stock reservation being created. As a result, the
+stock reservation will have no net impact on the listing's available
+stock.
+
+Note: It is recommended to place this action last in the transition's
+action list whenever possible.
+
+**Preconditions**:
+
+- The transaction must have a stock reservation
+- The stock reservation must be in either `proposed` or `pending` state
+
+**Parameters**:
```

Comment:
```suggestion
**Parameters**: -
```

Although looking at the rest of the file, this may not create a line break? (for example, look at https://github.com/sharetribe/flex-docs/blob/59e30409379d2b8471cb4ba8a121162e09fa3b6d/src/docs/references/transaction-process-actions/index.md#actionstripe-confirm-payment-intent, it shows `Parameters: - Configurations: -`).